music/src/music_vae/model.ts

```diff
@@ -1355,7 +1355,7 @@ class MusicVAE {
     }
     const randZs: tf.Tensor2D =
         tf.tidy(() => tf.randomNormal([numSamples, this.decoder.zDims]));
-    const outputTensors = this.decodeTensors(randZs, temperature);
+    const outputTensors = await this.decodeTensors(randZs, temperature);
```

Comment:
did we not need this before or was that a bug?

Reply:
I think it could result in the below dispose of randZs happening before they get used, though I never observed that behavior.